examples/cors/cors_middleware.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "github.com/savsgio/atreugo/v10"
+
+// CORS Setting
+var (
+	corsAllowHeaders     = "Content-Type, authorization"
+	corsAllowMethods     = "HEAD,GET,POST,PUT,DELETE,OPTIONS"
+	corsAllowOrigin      = "*"
+	corsAllowCredentials = "true"
+)
+
+func corsMiddleware(ctx *atreugo.RequestCtx) error {
+	// Avoid CORS middleware when you need
+	path := string(ctx.Path())
+	ignoredURL := map[string]bool{
+		"/no-cors": true,
+	}
+	if ignoredURL[path] {
+		return ctx.Next()
+	}
+
+	// Dynamic value
+	//corsAllowOrigin := string(ctx.URI().Scheme()) + "://" + string(ctx.Host())
```

Comment:
The origin must be got from `ctx.Request.Header.Peek("Origin")`

Reply:
I used this `ctx.Request.Header.Peek("Origin")` earlier in morning, but return empty, I will try again

Reply:
Don't worry because if you run your `index.html` directly in browser the origin is `null`, you could check it in the Chrome dev console.